repo-scripts/size-analysis/bundle-definitions/firestore.json

```diff
@@ -98,5 +127,102 @@
         ]
       }
     ]
+  },
+  {
+    "name": "Query Cursors",
+    "dependencies": [
+      {
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "app",
+            "imports": [
+              "initializeApp"
+            ]
+          }
+        ]
+      },
+      {
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "firestore",
+            "imports": [
+              "getFirestore",
+              "collection",
+              "doc",
+              "getDocs",
+              "query",
+              "orderBy",
+              "startAt",
+              "endBefore"
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "name": "Persistence",
+    "dependencies": [
+      {
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "app",
+            "imports": [
+              "initializeApp"
+            ]
+          }
+        ]
+      },
+      {
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "firestore",
+            "imports": [
+              "getFirestore",
+              "enableIndexedDbPersistence"
```

Comment:
Can you use `enableMultiTabIndexedDbPersistence`? That might provide more interesting data as it pulls in even more components.